collections/_content/donor-list.html

```diff
@@ -38,6 +38,8 @@ <h3 class="donor-list--level-label">
         <dt class="donor--name">{{ donor.label }}</dt>
         {%- if donor.location -%}
         <dd class="donor--location">{{ donor.location }}</dd>
+        {%- else -%}
+        <dd class="donor--none">Anonymous Location</dd>
```

Comment:
Weird one, but when there's not a location included, was getting an a11y error for improper use of a dt. So I added in this bit but hid it via CSS. Its extra labelling.